test/browser-tests/events/method-calls.js

```diff
@@ -174,6 +174,35 @@ test( 'component "on-" with ...arguments', t => {
 	component.fire( 'bar', 'bar', 100 );
 });
 
+test( 'component "on-" with additive ...arguments', t => {
+	t.expect( 7 );
+
+	const Component = Ractive.extend({
+		template: `<span id="test" on-click="foo:'foo', 42">click me</span>`
+	});
+
+	const ractive = new Ractive({
+		el: fixture,
+		template: '<Component on-foo="foo(\'fooarg\', ...arguments)" on-bar="bar(\'bararg\', ...arguments)"/>',
```

Comment:
I used template strings when I tried fixing this. No need to escape the quotes when written this way.

```js
template: `
  <component ... />
`,
```